Result/IndicesResult.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Result;
+
+/**
+ * This class is able to iterate over raw result.
+ */
+class IndicesResult
+{
+    /**
+     * @var array
+     */
+    private $rawData;
+
+    /**
+     * @param array $rawData
+     */
+    public function __construct(array $rawData)
+    {
+        $this->rawData = $rawData;
+    }
+
+    /**
+     * @return array
+     */
+    public function getTotal()
```

Comment:
Use an array for types, instead of using `func_get_args()`. Because we will be missing docBlock and users could become confused. Also for `getFailed`, `getSuccessful`